raiden/network/transport/matrix/transport.py

```diff
@@ -1138,81 +1133,90 @@ def _handle_member_join(self, room: Room) -> None:
                 partner_addresses
                 and self._address_mgr.get_address_reachability(partner_addresses[0])
                 == AddressReachability.REACHABLE
+                and self._capability_usable(
+                    Capabilities.WEBRTC,
+                    self._address_mgr.get_address_capabilities(partner_addresses[0]),
+                )
```

Comment:
Nitpick: I'd say move those conditions into variables. Such multiline ifs are pretty difficult to read IMO.